### PR TITLE
Feature/xivy 5007 docusign

### PR DESCRIPTION
**Pull request by ivy-rew**

### CODE REVIEW

File: market/connector/docusign/README.md [hunk 1]
```diff
@@ -1 +1,17 @@
-Integrate eSignatures into your application in minutes. DocuSign's secure and award-winning eSignature makes requesting signatures, automating forms, and tracking documents directly from your app easy.
\ No newline at end of file
+Integrate eSignatures into your application in minutes. DocuSign's secure and award-winning eSignature makes requesting signatures, automating forms, and tracking documents directly from your app easy.
+
+## Setup
+
+1. Create a free DocuSign developer account: https://account-d.docusign.com/#/username
+1. Create a new `application`: https://admindemo.docusign.com/authenticate?goTo=apiIntegratorKey
```
Comment (ivy-lli) [inline market/connector/docusign/README.md:6]: Seems to work, but I didn't know that this works if every numbered step is 1 :)

File: market/connector/uipath/meta.json [hunk 2]
```diff
@@ -6,22 +6,16 @@
 		{
 			"id": "project-resource",
 			"data": {
-				"resources": [
-					{
-						"source": "demo/uiPathDemo/src/com/uipath/orchestrator/feature/TenantHeaderFeature.java",
```
Comment (ivy-lli) [inline market/connector/uipath/meta.json:11]: if you don't copy those any longer to the project then I think you can also delete the resources itself?

**Review by ivy-lli — APPROVED** [review]
Looks good for me. But I think you can delete the uiPath features now, if you're not longer copy them to the project?

File: market/connector/uipath/meta.json [hunk 3]
```diff
@@ -6,22 +6,16 @@
...
-						"source": "demo/uiPathDemo/src/com/uipath/orchestrator/feature/FeatureConfig.java",
-						"path": "src/com/uipath/orchestrator/feature/FeatureConfig.java"
-					},
+				"directories": [
 					{
-						"source": "demo/uiPathDemo/src/com/uipath/orchestrator/feature/TokenStore.java",
-						"path": "src/com/uipath/orchestrator/feature/TokenStore.java"
+						"source": "demo/uiPathDemo/src/com/uipath/orchestrator/feature",
+						"includes": [ 
+							"TenantHeaderFeature.java",
```
Comment (ivy-rew) [inline market/connector/uipath/meta.json:13]: I still use these UiPath features (e.g. TenantHeader) @ivy-lli  : 
but the copy mechanism has become easier -> now we can declare full directories as shown here.

File: market/connector/docusign/README.md (hunk 1 above)
Comment (ivy-rew) [reply market/connector/docusign/README.md:6]: It's a markdown feature: and a quite handy one if you don't want to spend to much time when adding new points.
some editors, such as my favourite offline editor Hardoopad  https://github.com/rhiokim/haroopad , do also preview and render such numbered lists.

Comment (ivy-lli) [reply market/connector/docusign/README.md:6]: VS Code can preview too. But this one looks cool too 👍

**Pull request merged**